build fails look real:
```
======================================================================
ERROR: test_report_with_exclude_disabled (casexml.apps.stock.tests.test_logistics_consumption.LogisticsConsumptionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "corehq/ex-submodules/casexml/apps/stock/tests/test_logistics_consumption.py", line 95, in test_report_with_exclude_disabled
    self.create_transactions(self.domain.name)
  File "corehq/ex-submodules/casexml/apps/stock/tests/test_logistics_consumption.py", line 50, in create_transactions
    txn.save()
  File "corehq/ex-submodules/casexml/apps/stock/models.py", line 77, in save
    force_insert=force_insert, force_update=force_update, using=using, update_fields=update_fields
  File "/vendor/lib/python2.7/site-packages/django/db/models/base.py", line 589, in save
    force_update=force_update, update_fields=update_fields)
  File "/vendor/lib/python2.7/site-packages/django/db/models/base.py", line 626, in save_base
    update_fields=update_fields, raw=raw, using=using)
  File "/vendor/lib/python2.7/site-packages/django/dispatch/dispatcher.py", line 198, in send
    response = receiver(signal=self, sender=sender, **named)
  File "corehq/ex-submodules/casexml/apps/stock/signals.py", line 19, in stock_transaction_post_save_signal_receiver
    update_stock_state_for_transaction(instance)
  File "corehq/ex-submodules/casexml/apps/stock/signals.py", line 133, in update_stock_state_for_transaction
    state.save()
  File "/vendor/lib/python2.7/site-packages/django/db/models/base.py", line 589, in save
    force_update=force_update, update_fields=update_fields)
  File "/vendor/lib/python2.7/site-packages/django/db/models/base.py", line 617, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/vendor/lib/python2.7/site-packages/django/db/models/base.py", line 698, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/vendor/lib/python2.7/site-packages/django/db/models/base.py", line 731, in _do_insert
    using=using, raw=raw)
  File "/vendor/lib/python2.7/site-packages/django/db/models/manager.py", line 92, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/django/db/models/query.py", line 921, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/vendor/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 921, in execute_sql
    cursor.execute(sql, params)
  File "/vendor/lib/python2.7/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/vendor/lib/python2.7/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/vendor/lib/python2.7/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
IntegrityError: null value in column "last_modified_date" violates not-null constraint
DETAIL:  Failing row contains (9, stock, eaa09883c52d497190376fc0420b0e7e, 7ba81b01fbb04a319746b7cead4df8f4, 10.00000, null, null, null, 13, 484ef00d5e614eb8b0f97d558b540a80).
----------------------------------------------------------------------
```
and similar

--- a/corehq/apps/domain/tests/test_delete_domain.py
+++ b/corehq/apps/domain/tests/test_delete_domain.py
@@ -23,7 +23,8 @@ class TestDeleteDomain(TestCase):
             type='balance',
             domain=domain_name,
             form_id='fake',
-            date=datetime.utcnow()
+            date=datetime.utcnow(),
+            server_date=datetime.utcnow(),
         )
 
         StockTransaction.objects.create(

--- a/corehq/ex-submodules/casexml/apps/stock/tests/test_logistics_consumption.py
+++ b/corehq/ex-submodules/casexml/apps/stock/tests/test_logistics_consumption.py
@@ -1,5 +1,7 @@
 import uuid
 from decimal import Decimal
+
+from datetime import datetime
 from django.test import TestCase
 from casexml.apps.case.models import CommCareCase
 from casexml.apps.stock.models import StockReport, StockTransaction
@@ -35,6 +37,7 @@ class LogisticsConsumptionTest(TestCase):
         report = StockReport.objects.create(
             form_id=uuid.uuid4().hex,
             date=ago(2),
+            server_date=datetime.utcnow(),
             type=const.REPORT_TYPE_BALANCE,
             domain=domain
         )
@@ -52,6 +55,7 @@ class LogisticsConsumptionTest(TestCase):
         report2 = StockReport.objects.create(
             form_id=uuid.uuid4().hex,
             date=ago(1),
+            server_date=datetime.utcnow(),
             type=const.REPORT_TYPE_BALANCE,
             domain=domain
         )

--- a/corehq/ex-submodules/casexml/apps/stock/tests/test_stock_transaction.py
+++ b/corehq/ex-submodules/casexml/apps/stock/tests/test_stock_transaction.py
@@ -1,5 +1,7 @@
 from decimal import Decimal
 import uuid
+
+from datetime import datetime
 from django.test import TestCase
 from casexml.apps.case.models import CommCareCase
 from casexml.apps.stock.models import StockTransaction, StockReport
@@ -33,6 +35,7 @@ class StockTransactionTests(TestCase):
         report = StockReport.objects.create(
             form_id=uuid.uuid4().hex,
             date=ago(1),
+            server_date=datetime.utcnow(),
             type=const.REPORT_TYPE_BALANCE
         )
 

--- a/custom/ewsghana/tests/handlers/utils.py
+++ b/custom/ewsghana/tests/handlers/utils.py
@@ -43,6 +43,7 @@ class EWSScriptTest(EWSTestCase, TestScript):
         report = StockReport(
             form_id=xform._id,
             date=(now - datetime.timedelta(days=10)).replace(second=0, microsecond=0),
+            server_date=now,
             type='balance',
             domain=TEST_DOMAIN
         )

--- a/custom/ewsghana/tests/test_reminders.py
+++ b/custom/ewsghana/tests/test_reminders.py
@@ -22,13 +22,15 @@ from custom.ewsghana.utils import prepare_domain, bootstrap_user, bootstrap_web_
 TEST_DOMAIN = 'ews-reminders-test-domain'
 
 
-def create_stock_report(location, products_quantities, date=datetime.utcnow()):
+def create_stock_report(location, products_quantities, date=None):
+    date = date or datetime.utcnow()
     sql_location = location.sql_location
     report = StockReport.objects.create(
         form_id='ews-reminders-test',
         domain=sql_location.domain,
         type='balance',
-        date=date
+        date=date,
+        server_date=datetime.utcnow()
     )
     for product_code, quantity in products_quantities.iteritems():
         StockTransaction(

--- a/custom/ewsghana/tests/test_reminders.py
+++ b/custom/ewsghana/tests/test_reminders.py
@@ -30,7 +30,7 @@ def create_stock_report(location, products_quantities, date=None):
         domain=sql_location.domain,
         type='balance',
         date=date,
-        server_date=datetime.utcnow()
+        server_date=date
     )
     for product_code, quantity in products_quantities.iteritems():
         StockTransaction(

--- a/custom/ilsgateway/tanzania/handlers/la.py
+++ b/custom/ilsgateway/tanzania/handlers/la.py
@@ -30,9 +30,11 @@ class LossAndAdjustment(KeywordHandler):
             self.respond(LOSS_ADJUST_BAD_FORMAT)
             return True
 
+        now = datetime.utcnow()
         report = StockReport.objects.create(
             form_id='ilsgateway-xform',
-            date=datetime.utcnow(),
+            date=now,
+            server_date=now,
             type='balance',
             domain=self.domain
         )

--- a/custom/ilsgateway/utils.py
+++ b/custom/ilsgateway/utils.py
@@ -74,13 +74,15 @@ def make_loc(code, name, domain, type, metadata=None, parent=None):
     return loc
 
 
-def create_stock_report(location, products_quantities, date=datetime.utcnow()):
+def create_stock_report(location, products_quantities, date=None):
     sql_location = location.sql_location
+    date = date or datetime.utcnow()
     report = StockReport.objects.create(
         form_id='test-form-id',
         domain=sql_location.domain,
         type='balance',
-        date=date
+        date=date,
+        server_date=date,
     )
     for product_code, quantity in products_quantities.iteritems():
         StockTransaction(

